ci/computecpp.filter

```diff
@@ -28,3 +28,4 @@ usm
 vector_api
 vector_constructors
 vector_swizzles
+optional_kernel_features
```

Comment:
Do you maybe have output from a run of ComputeCpp or is this just to play it safe and not risk breaking CI?

Reply:
[Here](https://github.com/KhronosGroup/SYCL-CTS/runs/5209986322?check_suite_focus=true) output from a run of ComputeCpp before I apply filter. On line 2791 warning messages about unknown attributes, which is why I decided to put test in the filter.

But right now I also noticed other errors like `no known conversion from 'cl::sycl::nd_item<1>' to 'sycl::item<1>'` on line 2814. Am I wrong with the lambda argument? Do I have to use `nd_item<1>` instead of `item<1>` on line 41 in `kernel_features_mismatched_nd_range_exception.cpp`?